src/scc_hypervisor_collector/api/uploader.py

```diff
@@ -5,7 +5,59 @@
 collected from the specified backends to the SCC using the provided
 credentials.
 """
+import json
+import logging
+import gzip
+from typing import Dict
+from importlib_metadata import version
+import requests
+from requests.exceptions import RequestException
+from .configuration import SccCredsConfig
```

Comment:
I usually try to put a blank line separating the "standard library" imports from the local application imports...